api/server/router/grpc/grpc.go

```diff
@@ -14,9 +15,12 @@ type grpcRouter struct {
 
 // NewRouter initializes a new grpc http router
 func NewRouter(backends ...Backend) router.Router {
+	opts := []grpc.ServerOption{grpc.UnaryInterceptor(grpcerrors.UnaryServerInterceptor), grpc.StreamInterceptor(grpcerrors.StreamServerInterceptor)}
+	server := grpc.NewServer(opts...)
```

Comment:
Wondering why you used the intermediate `opts` here (as it's not currently customisable); perhaps

```suggestion
	server := grpc.NewServer(
		grpc.UnaryInterceptor(grpcerrors.UnaryServerInterceptor),
		grpc.StreamInterceptor(grpcerrors.StreamServerInterceptor),
	)
```